packages/remix-react/components.tsx

```diff
@@ -1663,15 +1662,17 @@ export const LiveReload =
                       config.onOpen();
                     }
                   };
-                  ws.onclose = (error) => {
+                  ws.onclose = (event) => {
                     console.log("Remix dev asset server web socket closed. Reconnecting...");
```

Comment:
Should this log move into the check as well? We aren't actually reconnecting if not, right?

Reply:
oops, yep